packages/react-admin/src/mui/layout/AppBarMobile.js

```diff
@@ -33,39 +33,29 @@ const styles = {
     },
 };
 
-class AppBarMobile extends Component {
-    handleLeftIconButtonTouchTap = event => {
```

Comment:
weird that this wasn't used anymore...

Reply:
Agreed, but it's not and the button is working